packages/api/src/external/carequality/command/update-patient-discovery-status.ts

```diff
@@ -1,14 +1,16 @@
-import { Patient } from "@metriport/core/domain/patient";
+import { Patient, PatientDemographicsDiff } from "@metriport/core/domain/patient";
 import { getPatientOrFail } from "../../../command/medical/patient/get-patient";
 import { PatientModel } from "../../../models/medical/patient";
 import { executeOnDBTx } from "../../../models/transaction-wrapper";
 
 export async function updatePatientDiscoveryStatus({
   patient,
   status,
+  patientDemographicsDiff,
 }: {
   patient: Pick<Patient, "id" | "cxId">;
   status: "processing" | "completed" | "failed";
+  patientDemographicsDiff?: PatientDemographicsDiff;
```

Comment:
When you make the update to check whether or not we got new patient demo at the end of the hies i dont think you will need to do this anymore. Would probably just do it in memory but thats only an assumption

Reply:
No longer relevent.